CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## main / unreleased
 * [FEATURE] tempo-cli: support dropping multiple traces in a single operation [#4266](https://github.com/grafana/tempo/pull/4266) (@ndk)
+* [CHANGE] Disable gRPC compression in the querier and distributor. [#4429] (https://github.com/grafana/tempo/pull/4429) (@carles-grafana)
```

Comment:
Let's add some more details here.

> Disable gRPC compression in the querier and distributor for performance reasons.
If you would like to reenable we recommend snappy. Use the following settings:
```
querier:
  <whatever the path to the setting is>
distributor:
  <whatever the path to the setting is>
```